x-pack/test/functional/apps/lens/index.ts

```diff
@@ -11,18 +11,23 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
   const browser = getService('browser');
   const log = getService('log');
   const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
 
   describe('lens app', () => {
     before(async () => {
       log.debug('Starting lens before method');
       await browser.setWindowSize(1280, 800);
       await esArchiver.load('x-pack/test/functional/es_archives/logstash_functional');
-      await esArchiver.load('x-pack/test/functional/es_archives/lens/basic');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
+      );
```

Comment:
The test failure on the last run and it's screenshot, shows that for some reason the end time in the timepicker didn't get set to the required date/time (in this file `kibana/x-pack/test/functional/apps/lens/smokescreen.ts`);
```
✖ fail: lens app  lens smokescreen tests should create a pie chart and switch to datatable
               │       Error: expected '@timestamp per 30 days' to sort of equal '@timestamp per 3 hours'
               │       + expected - actual
               │ 
               │       -@timestamp per 30 days
               │       +@timestamp per 3 hours
```

I thought about adding code in the time_picker page object method setAbsoluteTimeRange (sp?) to check that it actually did get the time set correctly.  But decided instead to suggest this change to make the tests faster by setting the default time for the timepicker.  This eliminates the need to call `PageObjects.lens.goToTimeRange()`

```suggestion
      );
      // changing the timepicker default here saves us from having to set it in Discover (~8s)
      await PageObjects.timePicker.setDefaultAbsoluteRangeViaUiSettings();
      await kibanaServer.uiSettings.update({'dateFormat:tz': 'UTC', });
```

Reply:
Using the timepicker here requires a couple other changes;

```
diff --git a/x-pack/test/functional/apps/lens/index.ts b/x-pack/test/functional/apps/lens/index.ts
index 0c3cbce23bd..7b26640738b 100644
--- a/x-pack/test/functional/apps/lens/index.ts
+++ b/x-pack/test/functional/apps/lens/index.ts
@@ -7,20 +7,24 @@
 
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export default function ({ getService, loadTestFile }: FtrProviderContext) {
+export default function ({ getService, loadTestFile, getPageObjects }: FtrProviderContext) {
   const browser = getService('browser');
   const log = getService('log');
   const esArchiver = getService('esArchiver');
   const kibanaServer = getService('kibanaServer');
+  const PageObjects = getPageObjects(['timePicker',]);
 
   describe('lens app', () => {
     before(async () => {
       log.debug('Starting lens before method');
       await browser.setWindowSize(1280, 800);
-      await esArchiver.load('x-pack/test/functional/es_archives/logstash_functional');
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
       await kibanaServer.importExport.load(
         'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
       );
+      // changing the timepicker default here saves us from having to set it in Discover (~8s)
+      await PageObjects.timePicker.setDefaultAbsoluteRangeViaUiSettings();
+      await kibanaServer.uiSettings.update({'dateFormat:tz': 'UTC', });
     });
 
     after(async () => {
@@ -28,6 +32,7 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       await kibanaServer.importExport.unload(
         'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
       );
+      await PageObjects.timePicker.resetDefaultAbsoluteRangeViaUiSettings();
     });
 
     describe('', function () {
```

Reply:
also note that a couple of lens tests load other esArchives, which wipes out the defaultTime setting change made in the lens/index file.  So if you wanted to remove the calls to the timepicker in those tests, you would have to add the call to 
`await PageObjects.timePicker.setDefaultAbsoluteRangeViaUiSettings();` in those tests after the esArchive load.